components/core/src/clp/streaming_archive/writer/utils.cpp

```diff
@@ -49,16 +51,18 @@ auto split_file_and_archive(
     auto has_ts_pattern = encoded_file.has_ts_pattern();
     auto orig_file_id = encoded_file.get_orig_file_id();
     auto split_ix = encoded_file.get_split_ix();
+    auto const end_msg_idx = encoded_file.get_end_msg_idx();
```

Comment:
If we're making the variables in `split_file` `const`, let's do the same here for consistency.